doc/releases/migration-guide-4.1.rst

```diff
@@ -204,6 +204,15 @@ DAC
 Enhanced Serial Peripheral Interface (eSPI)
 ===========================================
 
+* Renamed the devicetree property ``io_girq`` to ``io-girq``.
+* Renamed the devicetree property ``vw_girqs`` to ``vw-girqs``.
+* Renamed the devicetree property ``pc_girq`` to ``pc-girq``.
+* Renamed the devicetree property ``poll_timeout`` to ``poll-timeout``.
+* Renamed the devicetree property ``poll_interval`` to ``poll-interval``.
+* Renamed the devicetree property ``consec_rd_timeout`` to ``consec-rd-timeout``.
+* Renamed the devicetree property ``sus_chk_delay`` to ``sus-chk-delay``.
+* Renamed the devicetree property ``sus_rsm_interval`` to ``sus-rsm-interval``.
```

Comment:
heya, afraid these will have to go in 4.2, late in the RC to keep changing these, sorry

Reply:
yeah not a fan of having the 4.2 file in 4.1 to be totally honest, I think it's confusing, but that's what we decided to do... shrug